Error 2020-11-21 16:55:35: Jupyter Extension (Error in Failed to update settings.json, method:fixSettingsFile): [EntryNotFound (FileSystemError): Unable to read file '/home/code/python/.vscode/settings.json' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open '/home/code/python/.vscode/settings.json')
	at _handleError (/home/code/Downloads/VSCode-linux-x64/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:730:838)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async m.readFile (/home/code/.vscode/extensions/ms-toolsai.jupyter-2020.11.372831992/out/client/extension.js:49:824323)
	at async g.fixSettingsFile (/home/code/.vscode/extensions/ms-toolsai.jupyter-2020.11.372831992/out/client/extension.js:47:291968)] {
  code: 'FileNotFound',
  name: 'EntryNotFound (FileSystemError)'
}
> /usr/bin/python ~/.vscode/extensions/ms-toolsai.jupyter-2020.11.372831992/pythonFiles/pyvsc-run-isolated.py -c "import jupyter"
> /usr/bin/python ~/.vscode/extensions/ms-toolsai.jupyter-2020.11.372831992/pythonFiles/pyvsc-run-isolated.py -c "import jupyter"
> /usr/bin/python ~/.vscode/extensions/ms-toolsai.jupyter-2020.11.372831992/pythonFiles/pyvsc-run-isolated.py -c "import notebook"
> /usr/bin/python ~/.vscode/extensions/ms-toolsai.jupyter-2020.11.372831992/pythonFiles/pyvsc-run-isolated.py -c "import notebook"
